### PR TITLE
Update toolchain-android*.cmake

### PR DESCRIPTION
**Pull request by hham**
Update toolchain-android*.cmake for CMAKE_ANDROID_STANDALONE_TOOLCHAIN to be set accordingly.

### CODE REVIEW

File: standard/toolchain-android.cmake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 set(CMAKE_SYSTEM_NAME Android)
 set(CMAKE_SYSTEM_VERSION 21)
-find_path(CMAKE_ANDROID_STANDALONE_TOOLCHAIN sysroot/usr/lib/libandroid.so PATHS
-  /opt/android-standalone-toolchain
-)
+set(BUILD_ARM 1)
+set(BUILD_ANDROID 1)
```
Comment (jdonald) [inline standard/toolchain-android.cmake]: These two variables are Leap-isms that I don't think we necessarily want in the public repos (and we currently don't use them).

File: standard/toolchain-android32.cmake [hunk 2]
```diff
@@ -1,4 +1,11 @@
-if(NOT DEFINED CMAKE_ANDROID_ARCH_ABI)
-  set(CMAKE_ANDROID_ARCH_ABI armeabi)
-endif()
+set(CMAKE_ANDROID_ARCH_ABI armeabi)
+set(CMAKE_ANDROID_ARM_MODE ON)
```
Comment (jdonald) [inline standard/toolchain-android32.cmake]: Could keep `CMAKE_ANDROID_ARCH_ABI` conditional on whether it's not already defined, as @yeswalrus had it? I think the idea is that it should be easy to switch to another 32-bit architecture like armv7-a later if we decide to.

Are `CMAKE_ANDROID_ARM_MODE` and `-mfpu=neon` necessary here? Looks like we weren't using it before, and I checked the autowiring build scripts too.

**Comment by jdonald** [comment]
+1 for `CMAKE_ANDROID_STANDALONE_TOOLCHAIN` defaults and thus simpler build configuration steps.

File: standard/toolchain-android.cmake (hunk 1 above)
Comment (hham) [reply standard/toolchain-android.cmake]: Good catch!!

File: standard/toolchain-android32.cmake (hunk 2 above)
Comment (hham) [reply standard/toolchain-android32.cmake]: Ah..good point. I will recover the condition. I will remove 'mfpu=neon', too.

**Pull request merged**